GLTFSDK/Source/ExtensionsKHR.cpp

```diff
@@ -425,7 +435,9 @@ bool KHR::TextureInfos::TextureTransform::IsEqual(const Extension& rhs) const
         && this->offset == other->offset
         && this->rotation == other->rotation
         && this->scale == other->scale
-        && this->texCoord == other->texCoord;
+        && ((!this->texCoord && !other->texCoord) ||
```

Comment:
More readable like this?

```
&& static_cast<bool>(this->texCoord) == static_cast<bool>(other->textCoord)
&& (!static_cast<bool>(this->texCoord) || *this->texCoord == *other->texCoord)
```

Reply:
Um, is that even equivalent? I think it needs to be:

(texCoord == other->texCoord) || ((texCoord && other->texCoord) && (*texCoord == *other->texCoord))

Reply:
1. check that they're both set or both not set
2. either they're not set, or they're the same value

Reply:
Seems clear enough if you know it's a unique_ptr, no?